lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -5295,6 +5295,32 @@ namespace Js
         this->nativeHostPromiseContinuationFunctionState = state;
     }
 
+    void JavascriptLibrary::SetNativeHostPromiseRejectionTrackerCallback(HostPromiseRejectionTrackerCallback function)
+    {
+        this->nativeHostPromiseRejectionTracker = function;
+        this->hostShouldTrackPromiseRejections = true;
```

Comment:
Nit: set `nativeHostPromiseRejectionTracker` to nullptr upfront so that you can avoid this bool and just check if the function pointer is nullptr or not.

Reply:
Agreed - though I've now added a callbackState in the place of this.